src/test/java/org/folio/qm/extension/DatabaseInitializer.java

```diff
@@ -0,0 +1,33 @@
+package org.folio.qm.extension;
+
+import org.springframework.boot.test.util.TestPropertyValues;
+import org.springframework.context.ApplicationContextInitializer;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.context.event.ContextClosedEvent;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+public class DatabaseInitializer implements ApplicationContextInitializer<ConfigurableApplicationContext> {
+
+  @Override
+  public void initialize(ConfigurableApplicationContext configurableApplicationContext) {
+    JdbcDatabaseContainer<?> container = new PostgreSQLContainer<>("postgres:12-alpine").waitingFor(Wait.forListeningPort());
```

Comment:
testcontainers are good but only if there is docker available in the workstation running the tests
consider using https://github.com/zonkyio/embedded-database-spring-test